custom_components/frigate/camera.py

```diff
@@ -83,6 +83,8 @@ async def async_setup_entry(
 class FrigateCamera(FrigateMQTTEntity, Camera):  # type: ignore[misc]
     """Representation a Frigate camera."""
 
+    _attr_name = None
```

Comment:
Add a comment explaining why this is important / what this is for?

Reply:
Sounds good!